source/index.html.erb

```diff
@@ -15,90 +16,123 @@ description: Take and process online payments from your users - GOV.UK Pay is a
       </ol>
     </div>
     <div class="hero__content">
-        <div class="hero__body">
-            <h1 class="hero__title">Take online payments from your&nbsp;users</h1>
-            <div class="hero__inline-image">
-                <img src="/pay-product-page/images/pay-top-image.png" alt="" role="presentation">
-            </div>
-            <p class="hero__description">
-              If you work in the public sector, use GOV.UK Pay to take&nbsp;payments
-            </p>
-            <a href="/getstarted" role="button" draggable="false" class="govuk-button govuk-button--start button--start_white" data-module="govuk-button" data-click-events data-click-category="Hero" data-click-action="Button clicked">
-              Create a test account
-              <svg class="govuk-button__start-icon" xmlns="http://www.w3.org/2000/svg" width="17.5" height="19" viewBox="0 0 33 40" role="presentation" focusable="false">
-                <path fill="currentColor" d="M0 0h13l20 20-20 20H0l20-20z" />
-              </svg>
-            </a>
-            <span class="hero-alternative-action">
-              or <a href="/using-govuk-pay" data-click-events data-click-category="Hero" data-click-action="Internal link clicked">find out what GOV.UK Pay offers</a>
-            </span>
-        </div>
-        <div class="hero__aside-image">
-            <img src="/pay-product-page/images/pay-top-image.png" alt="Payment screen on mobile and dashboard with refund function." role="presentation">
+      <div class="hero__body">
+        <h1 class="hero__title">Take online payments from your users</h1>
+        <div class="hero__inline-image">
+          <%= image_tag 'pay-top-image.png', alt: 'Payment screen on mobile and dashboard with refund function.', role: 'presentation' %>
         </div>
+        <p class="hero__description">If you work in the public sector, use GOV.UK&nbsp;Pay to take payments</p>
```

Comment:
Sorry to nitpick here but we've lost the `&nbsp;` inside `take payments`. This has left a dangling word on the left hand side of the text block, which I think we want to avoid (I only spotted this comparing the local page with current production!).

Reply:
Fixed